src/store/actions.ts

```diff
@@ -3,4 +3,4 @@ import RootState from './RootState'
 
 const actions: ActionTree<RootState, RootState> = {}
 
-export default actions
+export default actions;
```

Comment:
Remove unwanted code changes.

Reply:
Removed unwanted code.
Thank you sir for reviewing my code.